__fixtures__/setup/jestSetUp.js

```diff
@@ -0,0 +1,25 @@
+// mock Logger
+jest.mock('../../lib/logger.js', () => {
+  return {
+    init: jest.fn(),
+    create: () => {
+      return {
+        info: jest.fn(),
+        warn: jest.fn(),
+        error: jest.fn(),
+        debug: jest.fn(),
+        fatal: jest.fn()
+      }
+    },
+    logTypes: {
```

Comment:
Do we need to repeat these constants here? Is there a way to reference it from `Logger`?

Reply:
I tried many different ways, it seems jest doesn't allow referencing out-of-scope variables. If you have heard of a method. would love to hear it.

Reply:
One solution i can think of is to separate the `logger` from `logTypes`

Reply:
I implemented it the solution i mentioned